newrelic-php-agent/values.yaml

```diff
@@ -1,11 +1,48 @@
-# Default values for simple-echo.
+# Default values for newrelic-php-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-image:
-  repository: chatwork/newrelic-php-agent
-  tag: latest
-  pullPolicy: IfNotPresent
-  command: []
-
-nameOverride: ""
-fullnameOverride: ""
\ No newline at end of file
+newrelic:
+  image:
+    repository: chatwork/newrelic-php-agent
+    tag: 0.0.1
+    pullPolicy: IfNotPresent
+    command: []
+
+  address: /tmp/.newrelic.sock
```

Comment:
@cw-niinuma 
`address` seems to be a value that should be passed to newrelic-daemon.
For example, if `127.0.0.1:31339` is specified for `address,` does NewRelic Daemon wait at that address?